redisinsight/api/src/modules/rdi/client/api.rdi.client.ts

```diff
@@ -31,6 +34,31 @@ export class ApiRdiClient extends RdiClient {
     return null;
   }
 
+  async dryRunJob(data: DryRunJobDto): Promise<DryRunJobResponseDto> {
+    const results: any = {};
+    try {
+      const transformations = await this.client.post(
+        RdiUrl.DryRunJob,
+        { input: data.input, job: data.job, test_output: false },
+      );
+      results.transformations = { status: DyRunJobStatus.Success, data: transformations.data };
+    } catch (e) {
+      results.transformations = { status: DyRunJobStatus.Fail, error: e.message };
+    }
+
+    try {
+      const commands = await this.client.post(
+        RdiUrl.DryRunJob,
+        { input: data.input, job: data.job, test_output: true },
+      );
+      results.commands = { status: DyRunJobStatus.Success, data: commands.data };
+    } catch (e) {
+      results.commands = { status: DyRunJobStatus.Fail, error: e.message };
+    }
+
+    return results;
+  }
+
```

Comment:
I prefer to have 2 separate methods to get transformations and commands. You can run them in parallel and compose results in a parent method `dryRunJob`. WDYT?

Reply:
Good idea, thank you